tidal/tools/chocolateyinstall.ps1

```diff
@@ -14,7 +14,7 @@ get-childitem $env:localappdata
 get-childitem $apmpath
 
 ### install SuperDirt
-$quarkinstall_path = $env:ChocolateyPackageFolder + '\tools\quarkinstall.sc'
+$quarkinstall_path = Join-Path $env:ChocolateyInstall  "\lib-bad\TidalCycles\tools\quarkinstall.sc"
```

Comment:
the `lib-bad` path looks suspicious to me. I want to verify that this PR works on my machine before we merge this.

Reply:
I can't find any docs on the `lib-bad` path. Just seems like a weird name.